src/pages/before-you-memo/index.es.md

```diff
@@ -0,0 +1,178 @@
+---
+title: 'Antes de utilizar memo()'
+date: '2021-02-23'
+spoiler: "Optimizaciones de renderizado que llegan naturalmente."
+---
+
+Hay muchos artículos escritos acerca de las optimizaciones de rendimiento de React. En general, si alguna actualización de estado es lenta, deberías: 
+
+1. Verificar que estás corriendo una compilación (build) de producción. (Las compilaciones (builds) de desarrollo son intencionalmente más lentas, incluso, en casos extremos en un orden de magnitud.)
+2. Verificar que no ha puesto el estado más arriba de lo necesario en el árbol. (Por ejemplo, poner la entrada del estado en un almacén centralizado podría no ser la mejor idea)
+3. Ejecutar React DevTools Profiler para ver qué se vuelve a renderizar, y envolver los subárboles más costosos con `memo()`. (Y agregar `useMemo()` donde sea necesario.)
+
+Este último paso es molesto, especialmente para los componentes intermedios, e idealmente un compilador lo haría por ti. En el futuro, probablemente.
```

Comment:
En el original dice «*in the future, it might*». Creo que usar «probablemente» afecta un poco el significado original, que no comunica el mismo nivel de confianza.

Otra opción:
```suggestion
Este último paso es molesto, especialmente para los componentes intermedios, e idealmente un compilador lo haría por ti. En el futuro, podría ser.
```